domains/secure.furqan.json

```diff
@@ -4,6 +4,11 @@
     "email": "frqnlove8765@gmail.com"
   },
   "record": {
-    "CNAME": "phonesme.onrender.com"
+    "CNAME": "phonesme.onrender.com",
+      "MX": [
+      "mx1.improvmx.com",
+      "mx2.improvmx.com"
+    ],
+    "TXT": "v=spf1 include:spf.improvmx.com ~all"
```

Comment:
You can't use CNAME with other records
either remove it or convert it to an A record